x-pack/plugins/enterprise_search/public/applications/shared/role_mapping/user_added_info.test.tsx

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+import { shallow } from 'enzyme';
+
+import { EuiText } from '@elastic/eui';
+
+import { UserAddedInfo } from './';
+
+describe('UserAddedInfo', () => {
+  const props = {
+    username: 'user1',
+    email: 'test@test.com',
+    roleType: 'user',
+  };
+
+  it('renders', () => {
+    const wrapper = shallow(<UserAddedInfo {...props} />);
+
+    expect(wrapper.find(EuiText)).toHaveLength(6);
```

Comment:
[optional] If you're passing dynamic text as props in here, I'd really suggest checking to make sure they got properly passed where you expect them to be.

If you want to make it quick, here's a few options:

Option A:
```js
const wapper = mount(...);
const text = wrapper.text();

expect(text).toContain('user1');
expect(text).toContain('test@test.com');
expect(text).toContain('user');
```

Option B:

TBH though, for this use case - check out [jest inline snapshots](https://jestjs.io/docs/snapshot-testing#inline-snapshots). I think it's a really solid use for it, and would probably be more useful of a test than checking EuiTexts.

```js
const wrapper = shallow(...);
expect(wrapper).toMatchInlineSnapshot();
```

^ run your tests and the above DOM tree will get automatically inserted inline, which should include dynamic props

Reply:
That's pretty cool Constance!

Reply:
Added to Google doc for tech debt after 7.14 FF

Reply:
This came in handy in a commit in my forthcoming PR. Removing from doc since it's done. Thanks for the tip!!